### ISSUE TITLE
PADNet loss scheme interpolation

### ISSUE THREAD

**Issue by NareshGuru77** [SimonVandenhende/Multi-Task-Learning-PyTorch#20]
PADNet paper mentions that the intermediate loss functions L1 to L4 are done by re-scaling the ground truth map to 1/4 resolution in page 3.

However, the implementation of the PADNet loss scheme upsamples the initial predictions to the img size before applying the loss. This seems to be in contrast to what the paper reports.
